src/voice_generator/gui.py

```
@@ -0,0 +1,128 @@
+                        f"Up to ~500 characters per Generate, or "
+                        f"{'∞' if CHAR_LIMIT is None else CHAR_LIMIT} characters per Stream"
+                    ),
+                )
+                with Row():
+                    voice: Dropdown = Dropdown(
+                        list(CHOICES.items()),
+                        value="af_heart",
+                        label="Voice",
+                        info="Quality and availability vary by language",
+                    )
+                    use_gpu: Dropdown = Dropdown(
```

Ruff (0.8.2)

Local variable `use_gpu` is assigned to but never used

Remove assignment to unused variable `use_gpu`

(F841)